requirements.txt

```diff
@@ -3,7 +3,8 @@ django-reversion~=6.0.0
 django-taggit~=6.1.0
 netjsonconfig @ https://github.com/openwisp/netjsonconfig/archive/refs/heads/1.3.tar.gz
 django-x509 @ https://github.com/openwisp/django-x509/archive/refs/heads/1.4.tar.gz
-django-loci @ https://github.com/openwisp/django-loci/archive/refs/heads/1.3.tar.gz
+# Todo: Make path https://github.com/openwisp/django-loci/archive/refs/heads/1.3.tar.gz before merge
+django-loci @ https://github.com/openwisp/django-loci/tarball/feature/191-new-ws-endpoint-for-all-location
```

Comment:
TODO: Remove this when https://github.com/openwisp/django-loci/pull/193 is merged.